AllTheThings.lua

```diff
@@ -9496,7 +9496,7 @@ local function RowOnEnter(self)
 		if reference.encounterID then
 			if app.Settings:GetTooltipSetting("encounterID") then GameTooltip:AddDoubleLine(L["ENCOUNTER_ID"], tostring(reference.encounterID)); end
 		end
-		if reference.factionID and app.Settings:GetTooltipSetting("factionID") then GameTooltip:AddDoubleLine(L["FACTION_ID"], tostring(reference.factionID)); end
```

Comment:
This is used by Faction Objects. (Such as the Warsong Outriders, not Horde/Alliance.)

Reply:
Ah, though for Horde/Alliance it does provide the correct text.